src/build/__main__.py

```diff
@@ -309,8 +311,9 @@ def main_parser() -> argparse.ArgumentParser:
             '    ',
         ),
         # Prevent argparse from taking up the entire width of the terminal window
-        # which impedes readability.
-        formatter_class=partial(argparse.RawDescriptionHelpFormatter, width=min(_max_terminal_width, 127)),
+        # which impedes readability. Also keep the description formatted.
+        formatter_class=partial(argparse.RawDescriptionHelpFormatter, width=min(_max_terminal_width, 127), **fmtopts),
+        **argopts,
```

Comment:
I prefer using a `partial` for this kind of thing - it has better typing support:

```py
make_parser = partial(argparse.ArgumentParser, ...)
if sys.version_info >= (3, 14):
    make_parser = partial(make_parser, color=True, suggest_on_error=True)
parser = make_parser()
```

Reply:
Oh, nice, hadn't thought of that.